src/main/java/at/hannibal2/skyhanni/features/misc/ChestValue.kt

```diff
@@ -0,0 +1,114 @@
+
+        for ((slot, item) in event.inventoryItems) {
+            value += EstimatedItemValue.getEstimatedItemPrice(item, mutableListOf()).first * item.stackSize
+            valueMap.addOrPut(item, EstimatedItemValue.getEstimatedItemPrice(item, mutableListOf()).first * item.stackSize)
+        }
+        itemValueTotal = value
+        var i = 0
+
+        itemEntryList.clear()
+        while (i<=3){
+            var highestValueEntry: Map.Entry<ItemStack, Double>? = null
+            var highestValue: Double? = null
+
+
+            for (entry in valueMap.entries) {
+                if (highestValue == null || entry.value > highestValue) {
+                    highestValueEntry = entry
+                    highestValue = entry.value
+                }
+            }
+            if (highestValueEntry != null) {
+                itemEntryList.add(highestValueEntry)
+                valueMap.remove(highestValueEntry.key)
+            }
+            i += 1
+        }
+    }
+
+    @SubscribeEvent(priority = EventPriority.HIGHEST)
+    fun onInventoryRender(event: DrawScreenAfterEvent) {
+        if (!LorenzUtils.inSkyBlock) return
+        if (!config.estimatedChestValue) return
+        val mc = Minecraft.getMinecraft()
+        val fontRenderer = mc.fontRendererObj
+        if (Minecraft.getMinecraft().currentScreen !is GuiChest) return
+        val gui = mc.currentScreen
+        val gui2: GuiChest = gui as GuiChest
+        val cc: ContainerChest = gui2.inventorySlots as ContainerChest
+        val guiName: String = cc.lowerChestInventory.displayName.unformattedText
+        if (guiName != "Large Chest" && guiName != "Chest") return
+        val accessorGui: AccessorGuiContainer? = gui as? AccessorGuiContainer
+        Minecraft.getMinecraft().textureManager.bindTexture(accessory_bag_overlay)
+        if (accessorGui != null) {
+            Utils.drawTexturedRect(
+                ((accessorGui.guiLeft + accessorGui.xSize) + 3).toFloat(),
+                accessorGui.guiTop.toFloat(),
+                80f,
+                149f,
+                0f,
+                80 / 256f,
+                0f,
+                149 / 256f,
+                GL11.GL_NEAREST
+            )
+
+            fontRenderer.drawString("Chest Value:", accessorGui.guiLeft + accessorGui.xSize + 10, accessorGui.guiTop + 10, Color.BLACK.rgb)
+            val string: String = EnumChatFormatting.BOLD.toString() + java.text.NumberFormat.getIntegerInstance().format(itemValueTotal.round(1)).toString()
+            fontRenderer.drawString(string, accessorGui.guiLeft + accessorGui.xSize + 10, accessorGui.guiTop + 10 + fontRenderer.FONT_HEIGHT, 16766208)
+            var i = 0
+            while(i<=itemEntryList.size - 1) {
+                val entry = itemEntryList[i]
+                GuiRenderUtils.renderItemStack(
+                    entry.key,
+                    accessorGui.guiLeft + accessorGui.xSize + 10,
```

Comment:
You could move `guiLeft`, `guiTop` and `xSize` into two local values to improve the readability